test/usecases/table/table.mod

```diff
@@ -0,0 +1,46 @@
+NEURON {
+    SUFFIX tbl
+    NONSPECIFIC_CURRENT i
+    RANGE e, g, gmax, v1, v2
+    GLOBAL k, d, c1, c2
+}
+
+PARAMETER {
+    e = 0
+    gmax = 0
+    k = .1
+    d = -50
+    c1 = 1
+    c2 = 2
+}
+
+ASSIGNED {
+    g
+    i
+    v
+    sig
+    v1
+    v2
+}
+
+BREAKPOINT {
+    sigmoid1(v)
+    g = gmax * sig
+    i = g*(v - e)
+}
+
+PROCEDURE sigmoid1(v) {
+    TABLE sig DEPEND k, d FROM -127 TO 128 WITH 155
+    sig = 1/(1 + exp(k*(v - d)))
+}
+
+FUNCTION example_function(arg) {
```

Comment:
If you want to match the previous theme: `quadratic` and `sinusoidal` might work as function/procedure names. Not sure there's much advantage to it.